modules/module-headings.php

```diff
@@ -1,351 +1,351 @@
-			//  - modules/after-the-deadline.php
-			//  - modules/custom-content-types.php
-			//  - modules/enhanced-distribution.php
-			//  - modules/json-api.php
-			//  - modules/latex.php
-			//  - modules/markdown.php
-			//  - modules/post-by-email.php
-			//  - modules/shortcodes.php
-			'Writing' =>_x( 'Writing', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Photos and Videos` tag:
-			//  - modules/carousel.php
-			//  - modules/photon.php
-			//  - modules/shortcodes.php
-			//  - modules/tiled-gallery.php
-			//  - modules/videopress.php
-			'Photos and Videos' =>_x( 'Photos and Videos', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Social` tag:
-			//  - modules/comment-likes.php
-			//  - modules/comments.php
-			//  - modules/gravatar-hovercards.php
-			//  - modules/likes.php
-			//  - modules/publicize.php
-			//  - modules/seo-tools.php
-			//  - modules/sharedaddy.php
-			//  - modules/shortcodes.php
-			//  - modules/shortlinks.php
-			//  - modules/subscriptions.php
-			//  - modules/widgets.php
-			'Social' =>_x( 'Social', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Appearance` tag:
-			//  - modules/custom-css.php
-			//  - modules/gravatar-hovercards.php
-			//  - modules/infinite-scroll.php
-			//  - modules/lazy-images.php
-			//  - modules/minileven.php
-			//  - modules/photon.php
-			//  - modules/seo-tools.php
-			//  - modules/shortcodes.php
-			//  - modules/widget-visibility.php
-			//  - modules/widgets.php
-			//  - modules/wordads.php
-			'Appearance' =>_x( 'Appearance', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Developers` tag:
-			//  - modules/json-api.php
-			//  - modules/pwa.php
-			//  - modules/sso.php
-			'Developers' =>_x( 'Developers', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Recommended` tag:
-			//  - modules/lazy-images.php
-			//  - modules/manage.php
-			//  - modules/minileven.php
-			//  - modules/monitor.php
-			//  - modules/photon.php
-			//  - modules/protect.php
-			//  - modules/publicize.php
-			//  - modules/related-posts.php
-			//  - modules/sharedaddy.php
-			//  - modules/sitemaps.php
-			//  - modules/stats.php
-			'Recommended' =>_x( 'Recommended', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Centralized Management` tag:
-			//  - modules/manage.php
-			'Centralized Management' =>_x( 'Centralized Management', 'Module Tag', 'jetpack' ),
-
-			// Modules with `General` tag:
-			//  - modules/masterbar.php
-			'General' =>_x( 'General', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Mobile` tag:
-			//  - modules/minileven.php
-			'Mobile' =>_x( 'Mobile', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Traffic` tag:
-			//  - modules/sitemaps.php
-			//  - modules/wordads.php
-			'Traffic' =>_x( 'Traffic', 'Module Tag', 'jetpack' ),
-
-			// Modules with `Site Stats` tag:
-			//  - modules/stats.php
-			'Site Stats' =>_x( 'Site Stats', 'Module Tag', 'jetpack' ),
-		);
-	}
-	return $module_tags[ $key ];
-}
\ No newline at end of file
+<?php
```

Comment:
I don't understand why this file changes all the time, do someone know?

Reply:
This file changes when you run `yarn build`. Usually the changes are trivial and you can just exclude it from the patch.

@cbauerman - can you please drop changes to this file from this patch? Thanks.